BUILD_SHARED_LIBS for ATS 10 build breaks

```
08:58:09 zeus:(master)~/dev/apache/trafficserver/build$ ninja
[0/2] Re-checking globbed directories...
[1/39] Linking CXX executable src/traffic_via/traffic_via
FAILED: src/traffic_via/traffic_via
: && /bin/c++ -pthread -g  src/traffic_via/CMakeFiles/traffic_via.dir/traffic_via.cc.o -o src/traffic_via/traffic_via  -Wl,-rpath,/home/bcall/dev/apache/trafficserver/build/src/tscore:/home/bcall/dev/apache/trafficserver/build/src/tsutil:/home/bcall/dev/apache/trafficserver/build/lib/swoc:/home/bcall/dev/apache/trafficserver/build/lib/yamlcpp:  -ldl  src/tscore/libtscore.so  src/tsutil/libtsutil.so  lib/swoc/libswoc-1.5.5.so  /usr/lib64/libcrypto.so  /usr/lib64/libpcre.so  lib/yamlcpp/libyaml-cppd.so.0.8.0  -ldl  /usr/lib64/libresolv.so  /usr/lib64/libcap.so  /usr/lib64/libhwloc.so && :
/bin/ld: src/tscore/libtscore.so: undefined reference to `mutexAllocator'
collect2: error: ld returned 1 exit status
[2/39] Linking CXX executable src/traffic_top/traffic_top
```

The symbol is here:
```
./src/iocore/eventsystem/libinkevent.a
000000000000040f l     F .text  000000000000000b _GLOBAL__sub_I_mutexAllocator
0000000000000000 g     O .bss   0000000000000008 mutexAllocator
```

--- a/include/tscore/FrequencyCounter.h
+++ b/include/tscore/FrequencyCounter.h
@@ -24,7 +24,7 @@
 #pragma once
 
 #include <cstdint>
-#include "iocore/eventsystem/EventSystem.h"
+#include <tscore/ink_hrtime.h>
 
 class FrequencyCounter
 {

